### ISSUE TITLE
Issue with cluster-level resources

### ISSUE THREAD

**Issue by rcohenDBRS** [weaveworks/flux#1843]
I've been seeing this issue a lot lately:
```
ts=2019-03-19T18:15:52.144930335Z caller=loop.go:412 component=sync-loop err="executing HTTP request: 400 Bad Request parsing <cluster>:clusterrole/kube2iam: invalid service ID\n"
```
It causes `fluxctl sync` to return `Error: timeout`, but no other discernible issues.

**Comment by squaremo** [comment]
Are you using Weave Cloud (or fluxcloud)? I ask because this looks like it's a problem with the events the fluxd posts -- basically, that it constructs events that the receiver can't process.

**Comment by squaremo** [comment]
We are seeing this in our own infrastructure too.

**Comment by rcohenDBRS** [comment]
I am using fluxcloud. I was able to fix it by updating the `flux-sync` tag to the latest commit.

**Comment by 2opremio** [comment]
@rcohenDBRS I think this should be fixed by #1851

**Comment by acobaugh** [comment]
Still seem to be seeing this, even on 1.12.0.
